src/commcare_cloud/ansible/roles/keepalived/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Variables for role keepalived
+keepalived_auth_pass: "1111"
+keepalived_role: "MASTER"
+keepalived_router_id: "52"
+keepalived_shared_iface: "eth0"
+keepalived_shared_ip: "192.168.1.1"
+monit_process: "nginx"
+keepalived_priority: "200"
+keepalived_backup_priority: "100"
+script_user: "root"
+notify_script_path: "/etc/keepalived/notify_script"
+smtp_server: "smtp.gmail.com"
+sender_email: "lb-alert@dimagi.com"
+recepient_email: "tcl-server-standup@dimagi.com"
+sender_password: "{{ secrets.sender_email_pass}}"
```

Comment:
i'm not sure about alternatives. if you guys are ok we can have an email id created for notification.

Reply:
we can reuse the same email settings we have for HQ, so this can be left blank here and then in `<env>/public.yml` (or even in `all.yml`) we can have something like this:
```
keepalived_smtp_host = "{{ localsettings.EMAIL_SMTP_PORT }}"
keepalived_smtp_login = "{{ EMAIL_LOGIN }}"
keepalived_smtp_password = "{{ EMAIL_PASSWORD }}"
keepalived_sender_email = "{{ root_email }}"
keepalived_recepient_email = "{{ default_from_email }}"
```

Side note: these email setting could do with a cleanup so they aren't so scattered.